Adding margin at the bottom of Button

shram-laabh opened the pull request:
This pull request address following issue:
https://github.com/zulip/zulip-mobile/issues/5431

It is required that button "Send private message" have some margin with respect to the bottom of screen.

chrisbobbe commented:
Thanks! Please [fix up](https://zulip.readthedocs.io/en/latest/git/fixing-commits.html) your commit message so that it follows our [style](https://github.com/zulip/zulip-mobile/blob/main/docs/style.md), and add a `Fixes: ` line to the commit message and to your PR description.

chrisbobbe commented:
Also, could you please post a [screenshot](https://zulip.readthedocs.io/en/latest/git/pull-requests.html#step-3-open-the-pull-request)? Thanks!

shram-laabh commented:
> Also, could you please post a [screenshot](https://zulip.readthedocs.io/en/latest/git/pull-requests.html#step-3-open-the-pull-request)? Thanks!



> Also, could you please post a [screenshot](https://zulip.readthedocs.io/en/latest/git/pull-requests.html#step-3-open-the-pull-request)? Thanks!

Sure Chris, i've added screenshot, and updated description in both PR description as well as commit message. Kindly review.

chrisbobbe commented:
Thanks, @shram-laabh! Merged, with one nit fixed: I put an empty line between the summary line and the commit-message body.
